zad3lista2

```diff
@@ -3,41 +3,36 @@
 
 using namespace std;
 
-void wypelnianie(int n, *tab){
-    int i;
-
-for (i=0; i<n; i++)
-    {
-        // pobieramy dane
-        cout<< "Podaj id tosoba numer " << i+1 << endl;
-        cin>>tab->id;
-        cout<< "Podaj imie tosoba numer " << i+1 << endl;
-        cin>>tab->imie;
-        cout<< "Podaj nazwisko tosoba numer " << i+1 << endl;
-        cin>>tab->nazwisko;
-        cout<< "Podaj wiek tosoba numer " << i+1 << endl;
-        cin>>tab->wiek;
-
-
-
-
-}
-}
-typedef struct tosoba{
+ struct osoba{
 int id;
 char imie[20];
 char nazwisko[20];
 int wiek;
-}
+};
 
+void wypelnianie(int n, osoba *tab){
+    int i;
+    for (i=0; i<n; i++){
+    cout<< "Podaj id tosoba numer " << i+1 << endl;
```

Comment:
Zapomnialem sie odwolywac do  wskaznika tab  tak jak w 2 zadaniu bylo, takze to sie potem poprawi i bedzie gitara :D